/**:
  ros__parameters:
    # Master file
    master: GPS                       # set map that is taken as reference, options: GPS, SLAM

    # Registration GPS/Poses
    align_num_inter_ume: 500          # Number of interpolation points for Umeyama method (algorithm needs equivalent number of points)
    rs_num_controlPoints: 10          # Number of control points for rubber-sheeting

    # Pointcloud transformation
    transform_pcd: false              # transform corresponding point cloud map to laneletmap if osm selected as master (align and rubber-sheet, only 2D)
    save_ascii: true                 # Safe .pcd file as: true - as ASCII file, false - as binary file
    pcd_path: "PATH/TO/PCD_MAP"

    # GPS zero coordinate
    customZeroPoint: false            # false - use first GPS data as zero point, true - set custom zero point with zeroLat/zeroLong
    zeroLat: 45.61989206965495        # Zero point latitude
    zeroLong: 9.283415137392435       # Zero point longitude

    # Buffer-Growing - Parameters
    seg_len: 5.0                      # [m] maximum length of a segment, otherwise it's splitted
    pline_angle: 30.0                 # [°] maximum angle of two following linestring segments that the reference polyline is not splitted
    buffer_V: 2.5                     # [m] initial size of the buffer in vertical direction
    buffer_P: 3.0                     # [m] initial size of the buffer in longitudinal direction
    buffer_rad: 0.5                   # [m] initial radius of the buffer on its corners

    lim_angle: 20.0                   # [°] angle constant to normalize angles of match candidates, candidate ignored if exceeding the limit
    lim_length: 20.0                  # [m] length constant to normalize lengths of match candidates, candidate ignored if exceeding the limit
    lim_chord: 15.0                   # [m] chord constant to normalize chord lengths of match candidates, candidate ignored if exceeding the limit
    lim_poly: 10.0                    # [m] constant to normalize polygon area/sum(lengths) of match candidates, candidate ignored if exceeding the limit
    w_angle: 0.35                     # Weighting factor for angle difference between connections of first and last point of polylines
    w_length: 0.2                     # Weighting factor for length difference between polylines
    w_chord: 0.35                     # Weighting factor for chord difference between polylines
    w_poly: 0.1                       # Weighting factor for quotient between area of polygon enclosed by polylines and the sum of their lengths

    lim_tp: 0.8                       # minimum score of a match to be considered as true positive (otherwise false positive)
    lim_ref_pline: 1.5                # minimum length of reference polyline to be considered during evaluation of matching

    # Visualization
    viz_lanelet_centerline: false      # Vizualization of centerlines of lanelets

    # Analysis Output
    analysis_output_dir: output       # directory to save output data
    analysis_traj_matching: true      # Visualize Rubber-Sheeting (true => data is saved in directory to be plotted with python script in ~/analysis)
    analysis_matching: true           # Visualize street network matching (true => data is saved in directory to be plotted with python script in ~/analysis)
